watchpoints.c

```diff
@@ -213,44 +202,43 @@ watchpoints_read(struct file *file, char __user * user_buffer,
 		output_pointer = output + last_entry_offset;
 		printk(KERN_DEBUG "%s", output);
 		
+		bytes_read = 0;
```

Comment:
You introduced a bug here. bytes_read should be set to 0 before the outer loop. please correct this